docker/setup-compose.yml

```diff
@@ -13,7 +13,7 @@
 version: "3"
 services:
   site:
-    image: codedotorg/code-dot-org:1.5.4
+    image: codedotorg/code-dot-org:node-18.16.0
```

Comment:
looks like these entries still need to be updated to `codedotorg/code-dot-org:1.7.0`, too